source/_integrations/geonetnz_quakes.markdown

```diff
@@ -7,6 +7,7 @@ ha_category:
 ha_iot_class: Cloud Polling
 ha_release: 0.98
 ha_config_flow: true
+ha_quality_scale: platinum
```

Comment:
This value is automatically synced to the docs. It is fine to leave it in, however, it could have been committed to the `current` branch already without this change.